go-controller/pkg/node/node.go

```diff
@@ -478,6 +501,62 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	return err
 }
 
+//AddOrRemoveTaint adds or removes the network-unavailable taint on the node depending on the setTaint value
+func (n *OvnNode) AddOrRemoveTaint(setTaint bool) error {
+	action := ""
+	if setTaint {
+		action = "adding"
+	} else {
+		action = "removing"
+	}
+
+	node, err := n.Kube.GetNode(n.name)
+	networkUnavailableTaint := &kapi.Taint{Key: "k8s.ovn.org/network-unavailable", Value: "NoSchedule", Effect: "NoSchedule"}
```

Comment:
use the upstream taint: `corev1.TaintNodeNetworkUnavailable`

Reply:
I can't really do this, for some reason something else is removing this taint as soon as ovnkube-node adds this. I don't know what is removing this.